### DISCUSSION TITLE
Osu!Lazer not working with Pulse audio/ALSA

### DISCUSSION THREAD

**Discussion by presi300** [ppy/osu#13876] [Q&A]
The pulse audio server just does not show as an audio option. The sound works... from an audio device that isn't used by anything else and when it works it mutes all other audio/microphones. I haven't had this issue before or on another distro (i am currently using arch).

![Screenshot_20210713_234326](https://user-images.githubusercontent.com/50592132/125522221-a2c6929a-54a4-49f8-9148-0ebd4b2ef13d.png)

On the image the osu audio works but all other audio on that device (my headphones) is muted...

**Comment by smoogipoo** [comment]
Are you sure you have pulse running? Can't repro.

**Reply by presi300** [reply]
yes, pulseaudio is running

**Comment by huupoke** [comment]
osu! is using BASS, which only supports ALSA backend (no PulseAudio, no Pipewire). On Arch Linux, you can install `pulseaudio-alsa` to route all ALSA application to PulseAudio (remember to use the "Default" device and remove your `~/.asoundrc` file). This will also resolve the "muted audio when running other applications" problem, since using raw ALSA means direct access to your hardware (soundcard), and most onboard soundcards only supports one audio stream. This problem will be resolved once osu! switches to another audio library that support PulseAudio.

**Reply by bdach** [reply]
> once osu! switches to another audio library that support PulseAudio

Clarification: That should be an "if", not a "once". Switching to another audio backend is just not going to be a priority until stable is phased out. See [comment](https://github.com/ppy/osu-framework/issues/2784#issuecomment-800049091).
